src/Dfe.PlanTech.Web/Controllers/CheckAnswersController.cs

```diff
@@ -16,30 +16,78 @@ public class CheckAnswersController : BaseController<CheckAnswersController>
 {
     public const string PAGE_SLUG = "check-answers";
 
-    public CheckAnswersController(ILogger<CheckAnswersController> logger) : base(logger)
+    public const string INLINE_RECOMMENDATION_UNAVAILABLE_ERROR_MESSAGE =
+        "Unable to determine your recommendation. Please try again.";
+
+    private readonly IUser _user;
+    private readonly IGetSectionQuery _getSectionQuery;
+    private readonly IProcessCheckAnswerDtoCommand _processCheckAnswerDtoCommand;
+    private readonly IGetPageQuery _getPageQuery;
+    private readonly ICalculateMaturityCommand _calculateMaturityCommand;
+
+    public CheckAnswersController(IUser user, IGetSectionQuery getSectionQuery,
+        IProcessCheckAnswerDtoCommand processCheckAnswerDtoCommand, IGetPageQuery getPageQuery, ICalculateMaturityCommand calculateMaturityCommand,
+        ILogger<CheckAnswersController> logger) : base(logger)
     {
+        _user = user;
+        _getSectionQuery = getSectionQuery;
+        _processCheckAnswerDtoCommand = processCheckAnswerDtoCommand;
+        _getPageQuery = getPageQuery;
+        _calculateMaturityCommand = calculateMaturityCommand;
     }
 
     [HttpGet("{sectionSlug}/check-answers")]
-    public async Task<IActionResult> CheckAnswersPage(string sectionSlug, [FromServices] IUser user, [FromServices] IGetSectionQuery getSectionQuery, [FromServices] IProcessCheckAnswerDtoCommand processCheckAnswerDtoCommand, [FromServices] IGetPageQuery getPageQuery, CancellationToken cancellationToken = default)
+    public async Task<IActionResult> CheckAnswersPage(string sectionSlug, CancellationToken cancellationToken = default)
     {
         if (string.IsNullOrEmpty(sectionSlug)) throw new ArgumentNullException(nameof(sectionSlug));
+        
+        var ErrorMessage = TempData["ErrorMessage"]?.ToString() ?? null;
```

Comment:
`?? null` is unneeded here, and can we camelCase `errorMessage` please. Sorry to be pedantic!